Smart4SURE.xcodeproj/project.pbxproj

```diff
@@ -269,22 +262,19 @@
-		80240A0A1D46949500100F21 /* Smart4SUREOnboardingViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Smart4SUREOnboardingViewController.swift; sourceTree = "<group>"; };
-		80240A0B1D46949500100F21 /* Smart4SUREPermissionsViewController.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Smart4SUREPermissionsViewController.swift; sourceTree = "<group>"; };
 		80240A251D469A2D00100F21 /* BridgeAppSDK.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = BridgeAppSDK.xcodeproj; path = BridgeAppSDK/BridgeAppSDK.xcodeproj; sourceTree = "<group>"; };
 		80240A621D469A9800100F21 /* BridgeSDK.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = BridgeSDK.xcodeproj; path = BridgeAppSDK/BridgeSDK/BridgeSDK.xcodeproj; sourceTree = "<group>"; };
 		80240A771D469AA600100F21 /* ResearchKit.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = ResearchKit.xcodeproj; path = BridgeAppSDK/ResearchKit/ResearchKit.xcodeproj; sourceTree = "<group>"; };
 		80240A811D469AB600100F21 /* ZipZap.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = ZipZap.xcodeproj; path = BridgeAppSDK/ZipZap/ZipZap.xcodeproj; sourceTree = "<group>"; };
 		80240AAA1D469B1700100F21 /* openssl.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = openssl.xcodeproj; path = BridgeAppSDK/CMSSupport/openssl/openssl.xcodeproj; sourceTree = "<group>"; };
 		80240ACD1D46A38A00100F21 /* Base */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; name = Base; path = Smart4SURE/Base.lproj/LearnInfo.plist; sourceTree = "<group>"; };
-		80240ACF1D46A38A00100F21 /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Smart4SURE/Base.lproj/Onboarding.storyboard; sourceTree = "<group>"; };
-		8027A8C61D638E5B00DCF4D0 /* BridgeInfo-private.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = "BridgeInfo-private.plist"; path = "../../iOSPrivateProjectInfo/Smart4SURE/BridgeInfo-private.plist"; sourceTree = "<group>"; };
-		8027A8DD1D638FDF00DCF4D0 /* sure-pd3-parkinson.pem */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; name = "sure-pd3-parkinson.pem"; path = "../../../../iOSPrivateProjectInfo/Smart4SURE/sure-pd3-parkinson.pem"; sourceTree = "<group>"; };
+		8027A8C61D638E5B00DCF4D0 /* BridgeInfo-private.plist */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text.plist.xml; name = "BridgeInfo-private.plist"; path = "../iOSPrivateProjectInfo/Smart4SURE/BridgeInfo-private.plist"; sourceTree = SOURCE_ROOT; };
+		8027A8DD1D638FDF00DCF4D0 /* sure-pd3-parkinson.pem */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = text; path = "sure-pd3-parkinson.pem"; sourceTree = "<group>"; };
```

Comment:
No, I changed the `Certificates` group to point at the private repo. It wasn't finding the pem file so I changed both the private files to use pointers related to project rather than relative to the group.